spring-cloud-commons/src/test/java/org/springframework/cloud/client/loadbalancer/LoadBalancerAutoConfigurationTests.java

```diff
@@ -116,7 +113,8 @@ LoadBalancerClient loadBalancerClient() {
 
         @Override
         public ServiceInstance choose(String serviceId) {
-            return new DefaultServiceInstance(serviceId, serviceId, random.nextInt(40000), false);
+            return new DefaultServiceInstance(serviceId, serviceId, random.nextInt(40000),
+                    false, Collections.<String, String>emptyMap());
```

Comment:
I think `DefaultServiceInstance` needs to retain the original constructor signature as well as the new one. Hadn't noticed this before.